tests/security/apparmor/aa_status.pm

```diff
@@ -17,21 +17,35 @@
 # aa-status.
 # - Check if apparmor is active
 # - Run aa-status, check the output for strings about modules/profiles/processes
+#   and verify "aa-status" can "handle profile with name contains '('".
 # and strings enforced, complain, unconfined and loaded.
 # Maintainer: llzhao <llzhao@suse.com>
-# Tags: poo#36874, poo#44912
+# Tags: tc#1767574, poo#81727, poo#36874, poo#44912
 
 use base "consoletest";
 use strict;
 use warnings;
 use testapi;
 use utils;
 use services::apparmor;
+use apparmortest qw(create_a_test_profile aa_status_stdout_check);
 
 sub run {
+    my ($self) = @_;
+
     select_console 'root-console';
     services::apparmor::check_service();
     services::apparmor::check_aa_status();
+
+    # Verify "aa-status" can "handle profile with name contains '('"
+    my $testfile    = "/usr/bin/ls";
+    my $str_special = '\(test\)';
+    apparmortest::create_a_test_profile_name_is_special($testfile, $str_special);
```

Comment:
should be added one blank line before this link.

Reply:
> should be added one blank line before this line.

Sure. though it is not necessary IMO.